<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VstSelectionWidget</class>
 <widget class="QWidget" name="VstSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_manageVst">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-style:italic;&quot;&gt;To manage your VST instruments, click &amp;quot;VST Instruments...&amp;quot; under the Live menu.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_vstis"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Instance:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="comboBox_chan">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>L</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>O</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Channel:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_vstisettings">
       <property name="text">
        <string>Edit...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_instance">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Create a New Instance</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QComboBox" name="comboBox_loopback"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
